src/collections/lookup-map.ts

```diff
@@ -13,7 +13,7 @@ export class LookupMap {
         return near.storageHasKey(storageKey)
     }
 
-    get(key: Bytes): unknown | null {
+    get(key: Bytes): T | null {
```

Comment:
The issue is what this get is an object instead of `T`. For example, the CarSpec get will be `{id, color, price, engine}`, not `CarSpec{id, color, price, engine}`